src/runtime/contrib/random/mt_random_engine.cc

```diff
@@ -126,8 +126,10 @@ class RandomEngine {
     } else {
       runtime::NDArray local = runtime::NDArray::Empty(
           std::vector<int64_t>{data->shape, data->shape + data->ndim}, data->dtype, {kDLCPU, 0});
-      FillData(&local.ToDLPack()->dl_tensor, size);
-      runtime::NDArray::CopyFromTo(&local.ToDLPack()->dl_tensor, data);
+      DLManagedTensor* dmt = local.ToDLPack();
```

Comment:
DLManagedTensor is not needed, we can directly use local.operator->() to get the DLTensor pointer

Reply:
Yes, but `local.operator->()` returns pointer to const DLTensor (`const DLTensor*`). So we won't be able to use it in `FillData` method.

Reply:
we can use const_cast here for now

Reply:
Done. Thank you for your comment